frontend/pages/shopping-lists/_id.vue

```diff
@@ -868,7 +868,6 @@ export default defineComponent({
 
         // set a temporary updatedAt timestamp prior to refresh so it appears at the top of the checked items
         item.updateAt = new Date().toISOString();
-        item.updateAt = item.updateAt.substring(0, item.updateAt.length-1);
       }
```

Comment:
Does this comment become no longer relevant? I'm not 100% sure what's meant to be happening here tbh.

Reply:
Yeah that comment still matters. The checked-off items show their `updateAt` time, so if we don't do this the item looks weird until we get a backend response